### DISCUSSION TITLE
where can I

### DISCUSSION THREAD

**Discussion by CheneyWong** [protobuf-c/protobuf-c#672] [Q&A]
My proto contains the google/protobuf/Any.proto, when I  gen proto by protoc-c use cmd "protoc-c --c_out=./ Sdk.proto", I got Sdk.pb-c.h file, this file include another file  named "any.pb-c.h" and I cant find this file anywhere.  

I find a same name file at google/protobuf/any.pb.h , but not  any.pb-c.h

**Comment by mhazley** [comment]
I had to generate it by doing:

```
protoc --c_out=. google/protobuf/any.proto
```

Still trying to figure out the decode though.....

**Comment by lipnitsk** [comment]
This doesn't look like an issue with the project. If you are having usage questions feel free to browse example code in this or other projects, or ask on the mailing list (link in README)
